Example/Example/Models/UserRepository.swift

```
@@ -69,7 +69,7 @@ extension Date: Argo.Decodable {
             if let date =  try? dateString.date(format: DateFormat.iso8601(options: .withFullTime)).absoluteDate {
                 return pure(date)
             } else {
-                return .typeMismatch(expected: "Date", actual: j)
+                return pure(Date())
```

why this changed? It seems that we should throw an error here

---

Yes, i added that because the parsing was failing and I wanted to test quickly, I forgot to fix it afterwards